safe-room.lic

```diff
@@ -47,6 +57,14 @@ class SafeRoom
     give_and_take(settings.safe_room_id, settings.safe_room_give, settings.safe_room_take)
   end
 
+  def get_health_score(health_hash)
```

Comment:
I realized overnight that this could delay heading even when bleeding so I updated the check method to force healing if any of the injuries are bleeders